src/main/java/seedu/address/model/task/Description.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.task;
```

Comment:
Just want to check if there is a need for this class or can we store the description as a String within the Task?

Reply:
I thought it was simpler to follow what was already implemented so that it is easier to check whether the description is valid too